stack.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import unicode_literals
+from linked_list import LinkedList
+
+
+class Stack(object):
+    def __init__(self, iterable=None):
+        self.data = LinkedList(iterable) if iterable else LinkedList()
+
+    def pop(self):
+        return self.data.pop()
+
+    def push(self, value):
+        self.data.insert(value)
```

Comment:
You're missing a newline at the end of this line.  Pep 8 says every line should have a "line terminator".  Putting in a newline character proves that the line is over.